lib/plugins/slack/attachements.js

```diff
@@ -108,9 +108,16 @@ module.exports = function(dataCollection, slackOptions) {
       }
     }
 
+    let message = '';
+    if (resultBaseURL) {
+      const baseDir = storageManager.getRelativeBaseDir();
+      const pagePath = storageManager.pathFromRootToPageDir(url);
+      message = ` (<${resultBaseURL}/${baseDir}/${pagePath}|result>)`;
+    }
+
     attachments.push({
       color,
-      text: url,
+      text: url + message,
```

Comment:
Maybe if the message variable was called text, initiated to url and appended to inside the if-statement. Then the attachment object could consistently use shorthand syntax.